back-end/src/main/kotlin/Gneiss/PacketCompiler/DatabaseAccess/PacketDao.kt

```diff
@@ -31,4 +33,26 @@ class PacketDao(jsonSerializer: IJsonSerializer) : IPacketDao {
         }
         return ret
     }
+
+    override fun getAllKeys(): Set<String> {
+        val jedis = pool.getResource()
+        val allKeys: Set<String> = jedis.keys("*")
```

Comment:
I have some concerns about assuming that every key in the Redis db is a user. For the future the next team may want to use Redis to store stuff like user settings. What if we store every user with a key of `USER#{user-id}` and then we can scan for keys that begin with `USER#`

Reply:
Good idea, I implemented this with USER:{user-id} as that matched matched paradigms I found online when searching